### DISCUSSION TITLE
Repetitive tasks

### DISCUSSION THREAD

**Discussion by kdiamond1** [hassio-addons/addon-grocy#250] [Ideas]
Could you please add a repetitive feature for tasks?  For example, changing air filters every X days/months/years?  That would make this a complete home management center.  Looks great - thank you for your hard work.

**Comment by Jason-nz** [comment]
This addon just packages grocy for use with home assistant. You'll want to request that at https://github.com/grocy/grocy

**Comment by kdiamond1** [comment]
ah geez.  sorry.   Thanks.
